assets/extensions/examples/online-extensions.js

```diff
@@ -58,27 +57,27 @@ angular
 
           system_status_elem_mobile.append(system_status_elem.clone());
           system_status_elem_mobile.css('display', '');
```

Comment:
If its worth the time, I could probably eliminate all the jQuery stuff here & just use a template